src/driver/main.cpp

```diff
@@ -475,13 +475,15 @@ struct compiler
             {
                 if(is_offload_copy_set(p) and not co.offload_copy)
                 {
-                    std::cout << "MIGraphX program was likely compiled with offload_copy set, Try "
-                                 "passing "
-                                 "`--enable-offload-copy` if program run fails.\n";
+                    std::cout
```

Comment:
Should we use `std::cerr` here instead? Like the MLIR warnings

Reply:
it is supposed to be a warning not an error.